main/lsp/protocol.cc

```diff
@@ -48,8 +48,10 @@ CounterState mergeCounters(CounterState counters) {
 }
 
 void tagNewRequest(spd::logger &logger, LSPMessage &msg) {
-    msg.latencyTimer = make_unique<Timer>(logger, "task_latency");
-    msg.latencyTimer->setTag("canceled", "false");
+    // apparently make_unique doesn't like initializer lists, so I'm passing it a Timer object directly.
+    msg.latencyTimer = make_unique<Timer>(
+        Timer(logger, "task_latency",
+              {50, 100, 250, 500, 1000, 1500, 2000, 2500, 5000, 10000, 15000, 20000, 25000, 30000, 35000, 40000}));
```

Comment:
Most requests will be clustered in the first buckets, so I made those with a finer granularity than larger buckets. The maximum bucket size here was chosen semi-arbitrarily, and the code will add a catch-all bucket for requests with latency >40 seconds.